README.md

```diff
@@ -197,6 +198,63 @@ $ oc start-build example-quarkus-binary --from-dir=target
 And that's it. In a couple minutes you should have your Kogito Service application up and running using the same binaries
 built locally.
 
+### Build From File
+
+Kogito Operator can configure a build for you that will pull and build your application through the source to image (s2i)
+feature (available only on OpenShift). This kind of build consists to upload a Kogito service file, e.g. a dmn or a bpmn file
+to the OpenShift Cluster and trigger a new Source to Image build automatically.
+
+You can provide a single dmn, drl, bpmn, bpmn2 and properties files, the standalone file can be locally on the filesystem or on a Web Site, like GitHub:
+
+```bash
+# from filesystem
+$ kogito deploy-service example-dmn-quarkus --from-file  /tmp/kogito-examples/dmn-quarkus-example/src/main/resources/"Traffic Violation.dmn"
+File found /tmp/kogito-examples/dmn-quarkus-example/src/main/resources/Traffic Violation.dmn.
+...
+The requested file(s) was successfully upload to OpenShift, a build with this file(s) should now be running. To see the logs, run 'oc logs -f bc/example-dmn-quarkus-builder -n kogito'
```

Comment:
"upload**ed** to openshift". Should fix in docs, and source code.

Reply:
done.